### PR TITLE
Update balance according to CA spec

### PR DESCRIPTION
**Pull request by JanJoris**

### CODE REVIEW

File: definitions/general/descriptions.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+type: object
```
Comment (bmoelans) [inline definitions/general/descriptions.yaml]: what about language/translations?

File: definitions/general/rates.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  type: object
+  title: Rate
+  required:
+  - type
+  - rate
+  properties:
+    type:
+      type: string
+      enum:
+      - VOICE
```
Comment (bmoelans) [inline definitions/general/rates.yaml]: list will be larger no, VOICE_ON_NET, VOICE_SUPER_ON_NET
OK to model, but would wait on Comarch before implementing North in UWA

File: definitions/products/product.yaml [hunk 3]
```diff
@@ -7,17 +7,9 @@ required:
...
-    type: object
-    required:
-      - title
-      - description
-    properties:
-      title:
-        type: string
-      description:
-        type: string
+    $ref: '#/definitions/Descriptions'
```
Comment (bmoelans) [inline definitions/products/product.yaml]: as with general definitions, what about translations?

File: definitions/general/descriptions.yaml (hunk 1 above)
Comment (JanJoris) [reply definitions/general/descriptions.yaml]: It's the same one as in product, I did not change this. We get all translations from CA and have an accept-language header from the client. We can fill this in ourselves. And if there's no accept-language header we default to NL.

File: definitions/general/rates.yaml (hunk 2 above)
Comment (JanJoris) [reply definitions/general/rates.yaml]: True, we did notice that too, just waiting on the actual names.

File: definitions/products/product.yaml (hunk 3 above)
Comment (JanJoris) [reply definitions/products/product.yaml]: Same reply as above ;-)

**Pull request merged**